spec/javascript/e2e/ApplicationNewPage.e2e.js

```diff
@@ -138,6 +138,19 @@ describe('ApplicationNewPage', () => {
     await page.select('#form-preferences\\.0\\.naturalKey', `${TRUNCATED_FIRST_NAME},${TRUNCATED_LAST_NAME},${DOB_YEAR}-${DOB_MONTH}-${DOB_DAY}`)
     await utils.blurValidation(page, '#form-preferences\\.0\\.naturalKey')
 
+    // Check ability to switch between the 2
+    // options with the type of proof updating accordingly
```

Comment:
Added check here where if we manipulate the selects one way and then the other this bug should be tested